build_script.py

```diff
@@ -206,6 +206,11 @@ def build(args):
                 build_dir=build_dir,
                 foundation_build_dir=foundation_build_dir))
 
+        # Build the static library
+        run("mkdir -p {build_dir}/static".format(build_dir=build_dir))
```

Comment:
It looks like there are three locations in this patch that are building `{build_dir}/static` path. It seems worthwhile to extract a helper for producing this path, similar to the existing `core_foundation_build_dir` method.